packages/form-json-schema/LICENSE

```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2021-present Camunda Services GmbH
```

Comment:
Can't we make it open source?

Reply:
I used the same license as for the other packages.